boards/rpi-pico-2-arm/include/board.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Tom Hert <git@annsann.eu>
+ * SPDX-FileCopyrightText: 2025 HAW Hamburg
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+#pragma once
+
+/**
+ * @ingroup         boards_rpi_pico_2_arm
+ * @{
+ *
+ * @file
+ * @brief           Board specific definitions for the Raspberry Pi Pico 2
+ *
+ * @author          Tom Hert <git@annsann.eu>
+ */
+
+ #include "cpu.h"
+ #include "periph_conf.h"
+
+ #include "periph/gpio.h"
```

Comment:
```suggestion
#include "cpu.h"
#include "periph_conf.h"

#include "periph/gpio.h"
```

Visual Studio doing Visual Studio things?

Reply:
This time this can be blamed on Zed I guess (actually probably clangd)